packages/hds/src/components/GaugeSteps/GaugeSteps.tsx

```diff
@@ -0,0 +1,30 @@
+import { HTMLAttributes, forwardRef } from 'react';
+import Gauge, { GaugeProps } from '../Gauge/Gauge';
+import Typo from '../Typo';
+import { guageStepsCss, guageStepsNames } from './GaugeStep.css';
+
+export interface GauageStepsProps extends GaugeProps, HTMLAttributes<HTMLDivElement> {
```

Comment:
```suggestion
export interface GauageStepsProps extends GaugeProps {
```
GaugeProps 자체가 이미 HTLMAttributes 를 extends 하고 있기 때문에 한 번 더 extends 하실 필요 없습니다.